packages/client/src/components/PlaceholderPages/ContactUs/ContactUs.component.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import './ContactUs.styles.css';
+
+export const ContactUs = () => {
+  return (
+    <main className="contactUs-wrapper">
+      <img
+        className="contactUs-img"
+        src="/assets/images/contactUs.png"
+        alt="contactUs"
+      />
+      <div className="contact-info-container">
+        <div className="contact-info">
+          <h3>Contact Us</h3>
+          <div className="fashion-email">
+            <img
+              className="icon-email"
+              src="../../assets/icons/email.png"
+              alt="email"
+            />
+            {/* TODO: replace the email icon with proper icon once it is placed with figma 48 */}
+            <p>Fashionspree@fashion.com</p>
+          </div>
+          <p className="or">-OR-</p>
+          <form>
+            <input
+              className="contactUs-input"
```

Comment:
I don't see the form validation or the logging, but no jira or TODO either, so not sure about that part. 
Should be either done or noted for future as pr the guidelines 🙏🏽

Reply:
Yeah Im working on this part now

Reply:
You can merge this without validation an etc. and make a new jira